layouts/foot.html

```diff
@@ -0,0 +1,3 @@
+    </div>
```

Comment:
I understand this is present in api-docs, but we can improve that.

First I would move everything(except `head` tags) from head to `default.html`.
Extract `navs` to layouts
Delete `foot` layout
Properly close / set everything in `default.html` - remember to place `yield` correctly.